packages/shared/src/lib/strings.ts

```diff
@@ -13,3 +13,11 @@ export const checkLowercaseEquality = (
   value1: string,
   value2: string,
 ): boolean => value1?.toLowerCase() === value2?.toLowerCase();
+
+export const sortAlphabetically = (a: string, b: string): number => {
```

Comment:
Found out that the `localCompare` is quite worse when used for sorting.

Reply:
Nice, today I learned 👍

Reply:
Also, this is not doing the sorting, just comparing, so not a good name.

And MDN itself suggests to use built in [`compare` for this same purpose](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/compare).

Reply:
I doubt there would be extreme difference in terms of performance, but if we can do it without spending a lot of effort, I guess why not?

Reference: https://stackoverflow.com/a/52369951/5532217

The difference here between localCompare and non-locale is quite significant. Shouldn't hurt to be performant on little time spent.
<img width="704" alt="Screenshot 2024-05-14 at 10 35 26 AM" src="https://github.com/dailydotdev/apps/assets/13744167/6ba949de-4816-47df-ba87-57d4cedefbe0">

Reply:
In my experience, it's usually better to use standard way. Since more people will be familiar with it. But if you feel strongly about this, I'll not stand in the way, consider my comments in this thread non-blocking :)

Reply:
But can we at least try to rename the function to `compareAlphabetically` or `fastCompare` or similar? 🙏

Reply:
Updated here: https://github.com/dailydotdev/apps/pull/3093/commits/1a182beb51e8cc92bcf158ccaa7c818ae44ed49f - think it is much cleaner this way 🚀